extensions/database/module/scripts/index/database-source-ui.js

```diff
@@ -64,6 +64,10 @@ Refine.DatabaseSourceUI.prototype.attachUI = function(body) {
   this._elmts = DOM.bind(this._body);
   var self = this;
   
+  self._defaultDatabaseHost = "localhost";
+  self._defaultDatabaseType = $( "select#databaseTypeSelect" ).val();
+  if (self._defaultDatabaseType == "") { self._defaultDatabaseType = "mariadb"; }
```

Comment:
In which scenario would the defaultDatabaseType be empty?  On a brand new installation of OpenRefine?

Reply:
The only scenario where it would be empty is if a coder changes the HTML code.

But, since it can happen, I check it. What I would really want to do here, is add a preference, for both the IP (host) and DB-type. But that feature is not yet merged in master.

Regards,
   Antoine

Reply:
OK, understood.